README.md

```diff
@@ -25,8 +27,18 @@ Categories:
 
 ## Usage
 
-Using the terminal change the current directory to the directory with the clean_my_dir.py file. Run command:
+Using the terminal run the program and provide a path to the directory as an
+argument:
 
-```python clean_my_dir.py```
+```python clean_my_dir.py '\path\to\directory'```
```

Comment:
"/path/to/directory"
use double quotes so shell can expand strings and use / unix path separators. Your script should be able to handle both

Reply:
ok